mozci/console/commands/push.py

```diff
@@ -508,6 +576,44 @@ def send_emails(self, total, stats, error_line):
             ),
         )
 
+    def log_details(self, push, state, expected):
+        total = len(self.failures[push][state])
+        if not total:
+            return {"total": 0, "correct": 0, "wrong": 0, "missed_conflicting": 0}
+
+        missed_conflicting = []
+        differing = []
+        for group in self.failures[push][state].keys():
+            classifications_set = set(
+                [c for c, _ in push.group_summaries[group].classifications]
+            )
+            if len(classifications_set) != 1:
+                missed_conflicting.append(group)
```

Comment:
Nit: could you split this in two, one for missing and one for conflicting? Missing should be 0 in theory, as all failures on autoland should be classified by sheriffs

Reply:
Sure